cache/compression.go

```diff
@@ -98,3 +98,8 @@ func (s *snappyCache) Fetch(ctx context.Context, keys []string, opts ...Option)
 func (s *snappyCache) Name() string {
 	return s.next.Name()
 }
+
+func (s *snappyCache) Delete(_ context.Context, _ string) error {
+	// no-op
```

Comment:
Given most of these are wrappers around another Cache implementation, I think it would be better to at least delegate the deletion through to the underlying cache. It would be confusing to use snappyCache for example and suddenly deletes stop working.

Reply:
Fixed.